From 568a80667a7d09e22e1022d061ca391ff1132b62 Mon Sep 17 00:00:00 2001
From: Tal Yacobi <tyacobi@habana.ai>
Date: Sun, 08 Jun 2025 15:43:51 +0300
Subject: [PATCH] [SW-221571] drivers: set device init log to warning

Temporarily set the log level to warning after device_init()
since it's expected to fail until the next patch is applied.

This causes a CI failure in patches on other repos
that needs to be applied before the device tree changes
so the build won't break.

Change-Id: Ic8d41aa726523b2a3fcfd87dd74a9cc04647b182
Signed-off-by: Tal Yacobi <tyacobi@habana.ai>
---

diff --git a/include/zephyr/drivers/misc/common/common_utils.h b/include/zephyr/drivers/misc/common/common_utils.h
index 97fe05d..8b7b63a 100644
--- a/include/zephyr/drivers/misc/common/common_utils.h
+++ b/include/zephyr/drivers/misc/common/common_utils.h
@@ -224,7 +224,7 @@
 				    (const struct device *dev = DEVICE_DT_GET(d);                  \
 					int err = device_init(dev);                                \
 					if (err) {                                                 \
-						LOG_ERR("Failed deferred init of device: %s (%d)", \
+						LOG_WRN("Failed deferred init of device: %s (%d)", \
 							#d, err);                                  \
 				     }),                                                           \
 				    (NULL));                                                       \
